config/locales/views/talk/show/en.yml

```diff
@@ -0,0 +1,7 @@
+# See https://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
+
+en:
+  talk:
+    show:
+      welcome: "Hello Everyone! Welcome to Publiclab Talk Pad"
```

Comment:
Actually the shorter one line version I posted above should be all you need
for the page itself... The rest can go in the wiki. No h3 header needed --
we want to keep the vertical height minimal so the ether pad instance has
lots of space.

On Aug 12, 2016 1:53 PM, "Jitesh Kumar Jha" <notifications@github.com>
wrote:

> In config/locales/views/talk/show/en.yml
> <https://github.com/publiclab/plots2/pull/645#discussion_r74585178>:
>
> > @@ -0,0 +1,7 @@
> > +# See https://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
> > +
> > +en:
> > +  talk:
> > +    show:
> > +      welcome: "Hello Everyone! Welcome to Publiclab Talk Pad"
>
> I'm finalizing on this paragraph. Is this alright? :
>
> Welcome to the Public Lab Talk Pad.
> This page allows for synchronized editing by multiple people as you type.
> It can be used to help you:
>
>    - collaborate with fellow wiki editors,
>    - take notes with multiple people in real time, and
>    - log the changes, decisions and notes made about a wiki page, or a
>    topic. Learn more <http://publiclab.org/wiki/talk-pages>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/publiclab/plots2/pull/645/files/d12cbef0d1f61cd7356075bd8797ca8f0831107d#r74585178>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABfJ-qAtb-713SVhIRM-I27RcOIS9DGks5qfGzRgaJpZM4JZK7K>
> .
>